### PR TITLE
BAEL-3502 Arrays.deepEquals method examples.

### PR DESCRIPTION
**Pull request by AlexFlorinHriscu**

### CODE REVIEW

File: core-java-modules/pom.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
         <module>core-java-annotations</module>
         <module>core-java-arrays</module>
         <module>core-java-arrays-2</module>
+		<module>core-java-arrays-3</module>
```
Comment (JonCook) [inline core-java-modules/pom.xml]: Formatting is also off here.

File: core-java-modules/core-java-arrays-3/src/test/java/com/baeldung/arrays/deepequals/ArraysDeepEqualsUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
+package com.baeldung.arrays.deepequals;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
```
Comment (JonCook) [inline core-java-modules/core-java-arrays-3/src/test/java/com/baeldung/arrays/deepequals/ArraysDeepEqualsUnitTest.java]: Formatting/Indentation is off for this class.

File: core-java-modules/core-java-arrays-3/pom.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (JonCook) [inline core-java-modules/core-java-arrays-3/pom.xml]: According to eclipse formatter, the indentation is also off here. Thanks.

File: core-java-modules/pom.xml (hunk 1 above)
Comment (AlexFlorinHriscu) [reply core-java-modules/pom.xml]: I applied the formatter and now I see that all the file looks like being modified. Let me know if it is ok now.

File: core-java-modules/core-java-arrays-3/src/test/java/com/baeldung/arrays/deepequals/ArraysDeepEqualsUnitTest.java (hunk 2 above)
Comment (AlexFlorinHriscu) [reply core-java-modules/core-java-arrays-3/src/test/java/com/baeldung/arrays/deepequals/ArraysDeepEqualsUnitTest.java]: Done.

File: core-java-modules/core-java-arrays-3/pom.xml (hunk 3 above)
Comment (AlexFlorinHriscu) [reply core-java-modules/core-java-arrays-3/pom.xml]: Done

File: core-java-modules/pom.xml (hunk 1 above)
Comment (AlexFlorinHriscu) [reply core-java-modules/pom.xml]: Ok, I think now it is correct.

**Pull request merged**